test/tool/utils.js

```diff
@@ -0,0 +1,172 @@
+'use strict';
+
+import fs from 'fs';
+import path from 'path';
+import {Tensor, sizeOfShape} from '../../src/lib/tensor.js';
+import {transpose} from '../../src/transpose.js';
+
+
+const getRandomFunctions = {
+  float64: getRandomFloat64,
+};
+
+/**
+ * Convert data as required precision type.
+ * @param {Array<Number>} input
+ * @param {String} precisionType
+ * @return {Array<Number>}
+ */
+function getPrecisionData(input, precisionType) {
```

Comment:
Name sounds odd, because it sounds like it's getting information about the precision itself, rather than casting the data to a given precision. How about `getDataCastToPrecision` or `castDataToPrecision`?